routes/stage.routes.js

```diff
@@ -0,0 +1,15 @@
+import express from "express";
+import * as stageController from "../controllers/stage.controller.js";
+import { authenticateToken } from "../middlewares/authentication.js";
+
+const router = express.Router();
+
+router.use(authenticateToken); // todo: wrap in a general protected router
+
+router.post("/create", stageController.createStage);
+router.get("/:stageid", stageController.getStage);
+router.get("/tournament/:stageid", stageController.getTournamentStages);
```

Comment:
The parameter is `stageId` here. But, it's referenced as `tournamentId` in the `getTournamentStages` controller. That will error because `tournamentId` will be `undefined` in `req.params`.

Also, I think it would make more sense if put the route in `tournamentRouter`. Perhaps under the route: "/:tournamentId/stages":

http://localhost:3912/tournaments/1231455/stages